controllers/registerController.js

```diff
@@ -37,10 +55,18 @@ exports.postRegistraionInfo = (req, res, next) => {
                     message: 'Player added',
                     playerID: result.insertedId
             })
-            .end()
-            // don't call next()
-            // need to return something json because frontend expects to receive `json.
-            // no need send(), res.json does this. 
+            try {
+                transporter.sendMail(message)
```

Comment:
One more comment. 🙏
This function is a `async` function. Need to call with `await`. 
https://nodemailer.com/about/#example

We might have unexpected bugs, and `try/catch` can't catch the error because we're not "awaiting" `transporter.sendMail(message)` to be done.